packages/onboarding/src/scroll-to-top/index.tsx

```diff
@@ -0,0 +1,17 @@
+/**
+ * External dependencies
+ */
+import * as React from 'react';
+import { useLocation } from 'react-router-dom';
+
+const ScrollToTop: React.FunctionComponent = () => {
+	const { pathname } = useLocation();
+
+	React.useEffect( () => document.querySelector( '.components-modal__content' )?.scrollTo( 0, 0 ), [
```

Comment:
Could we pass the selector as a prop in to the component to increase the flexibility of this component?

`<ScrollToTop selector=".components-modal__content" />`